pkg/graph/schema.resolvers.go

```diff
@@ -810,6 +810,12 @@ func (r *subscriptionResolver) OnTaskListSectionLocksChanged(ctx context.Context
 	return resolvers.SubscribeTaskListSectionLockChanges(r.pubsub, modelPlanID, principal, ctx.Done())
 }
 
+// AutoLockTaskListSection is the resolver for the autoLockTaskListSection field.
+func (r *subscriptionResolver) AutoLockTaskListSection(ctx context.Context, modelPlanID uuid.UUID, section model.TaskListSection) (<-chan *model.TaskListSectionLockStatusChanged, error) {
+	principal := appcontext.Principal(ctx).ID()
+	return resolvers.AutoLockTaskListSection(r.pubsub, modelPlanID, section, principal, ctx.Done())
+}
```

Comment:
Rather than introducing a new subscription, would it be possible to perform this lock removal on disconnect in the other, existing subscription?

Reply:
Are you referring to onTaskListSectionLocksChanged?

If so, although the subscriptions are similar they have different purposes.
onTaskListSectionLocksChanged - Read only. Listens for all changes to task list section locks.
autoLockTaskListSection - Subscription acts as scoped acquisition. While opened, the subscription maintains a section lock reference. Changes to task list section locks are provided only as a convenience for FE.